### DISCUSSION TITLE
Sample for ResultWriter usage

### DISCUSSION THREAD

**Discussion by rpbissonnette** [openai/whisper#1584] [Show and tell]
It took me an embarrassing amount of time to work out how to call the ResultWriter.  Did I miss some example documentation?

Anyway, until someone can provide a nicely cleaned up example, here is some code that worked for me.

[ResultWriter-example.txt](https://github.com/openai/whisper/files/12284870/ResultWriter-example.txt)

**Comment by phineas-pta** [comment]
#1289

**Comment by brosano** [comment]
Thank you! I don't understand why there isn't an example of this, the example on the mainpage just transcribe using decode and give you the first 30s of audio. I lost several hours too just trying to do exactly this

Thanks again!
